helm/ako/templates/configmap.yaml

```diff
@@ -16,6 +16,8 @@ data:
   enableEVH: {{ .Values.AKOSettings.enableEVH | quote }}
   layer7Only: {{ .Values.AKOSettings.layer7Only | quote }}
   istioEnabled: {{ .Values.AKOSettings.istioEnabled | quote }}
+  vipPerNamespace: {{ .Values.AKOSettings.vipPerNamespace | quote }}
+  vcfCluster: {{ .Values.AKOSettings.vcfCluster | quote }}
```

Comment:
Is there a reason we want to expose this flag in the AKO Configmap?

Reply:
without a flag it would be difficult to test internally